src/router/routes.ts

```diff
@@ -44,8 +44,38 @@ export const routes = [
     }
   },
   {
-    path: '/restoration',
-    name: RouteNames.RESTORATION,
+    path: '/limitedRestorationToFull',
```

Comment:
Is there a possible shorter path (route) name?

Reply:
Yes, we can definitely shorten them, but then it means adding more complicated logic to filings-ui.  At present, it's nice and simple:

```javascript
if (applicationName === ApplicationTypes.EDIT_UI) {
  url = `${this.getEditUrl}${this.getIdentifier}/` + restorationType + `?restoration-id=${id}`
}
```

Reply:
The subtype-renaming conversation above did not identify a strong need to change anything, so THIS change is also OK (since the route name and path is the subtype name).